frontend/js/src/playlists/components/PlaylistMenu.tsx

```diff
@@ -179,6 +214,7 @@ function PlaylistMenu({
   const showSpotifyExportButton = spotifyAuth?.permission?.includes(
     "playlist-modify-public"
   );
+  const showAppleMusicExportButton = spotifyAuth;
```

Comment:
This does not look correct. should probably be using appelAuth instead of spotifyAuth?